json_object.c

```diff
@@ -377,8 +385,7 @@ static int json_object_object_to_json_string(struct json_object* jso,
 	}
 	if (flags & JSON_C_TO_STRING_SPACED)
 		return sprintbuf(pb, /*{*/ " }");
-	else
-		return sprintbuf(pb, /*{*/ "}");
+	return sprintbuf(pb, /*{*/ "}");
```

Comment:
Why not

    return sprintbuf(pb,  /*{*/ (flags & JSON_C_TO_STRING_SPACED) ? " }" :  "}" );

and completely remove the `if`?